### ISSUE TITLE
Add BatchNormalization to Critic

### ISSUE THREAD

**Issue by armenkasp** [JeffersonLab/SciOptControlToolkit#6]
Batch normalization critic model
